lib/session.js

```diff
@@ -63,8 +63,16 @@ Session.prototype.enumerateModules = function () {
   return this[modulesPromise];
 };
 
-Session.prototype.enumerateRanges = function (protection) {
-  return this[request]('process:enumerate-ranges', { protection: protection })
+Session.prototype.enumerateRanges = function (protection, options) {
+  options = options || {};
+  scope = options.scope || null;
+  var name = "process:enumerate-ranges";
+  var data = { protection: protection };
+  if (!scope) {
```

Comment:
We usually try to be very explicit and always use `scope !== null` instead of `!scope`; and basically only use `if (scope)` or `if (!scope)` when `scope` is a boolean.